src/Umbraco.Web/Strategies/RelateOnTrashHandler.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Linq;
+using Umbraco.Core;
+using Umbraco.Core.Auditing;
+using Umbraco.Core.Events;
+using Umbraco.Core.Models;
+using Umbraco.Core.Services;
+
+namespace Umbraco.Web.Strategies
+{
+    public sealed class RelateOnTrashHandler : ApplicationEventHandler
```

Comment:
Does this need to be in the Web project? There doesn't seem to be anything directly relating to web - if there's no web project dependencies here it should exist in the Core project since it's a data driven process (i.e. we'd still want these relations created if people are using our APIs outside of a web context)

I realize RelateOnCopyHandler is also in Web but it shouldn't be. In any case, I'll move that to core and move this in to core when I pull it in.